tests/fixtures/scenarios/completions/stacks/deploy/dev.yaml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://atmos.tools/schemas/atmos/atmos-manifest/1.0/atmos-manifest.json
+
+vars:
+  stage: dev
+
+import:
+  - catalog/station
```

Comment:
Do not use the weather station as a test scenario. We DoS attack the wttr.in service.

Reply:
in completions we do not do any terraform plan command. So nothing to worry about.